### DISCUSSION TITLE
dragging to create a sibling

### DISCUSSION THREAD

**Discussion by wordmuse** [freeplane/freeplane#613] [Help]
In previous versions of FP, if I dragged a node from outside a hierarchy, I could drag it to a sibling position or a child position. FP would visually show me which I was doing.

But recent versions have made it so I can only drag such a node to a child position - even though the visual cue that I am dragging to a sibling position remains.  

Can this be looked into and corrected?  Or if it's a setting that I somehow inadvertently changed, let me know that.

If necessary, I can provide a video of what I'm experiencing.

Thanks.
~ Bal

**Comment by macmarrum** [comment]
Hi Bal

The drag-to-sibling functionality works OK for me.
I am not aware of any setting that would affect drag-to-child/sibling.

Could you please specify the version of Freeplane you are using and in case of a pre-release, it's number?

That video might be helpful too.

Mac

**Comment by wordmuse** [comment]
Thanks for your response, Mac.
Version 10.1.4

I'll create the video a little later on today.

~ Bal

**Reply by macmarrum** [reply]
> Version 10.1.4

Version 1.10.4 is still in preview. If you're using anything else than the latest preview (`pre11` at the moment), please upgrade and retry: https://sourceforge.net/projects/freeplane/files/freeplane%20preview/

**Comment by wordmuse** [comment]
I'll make sure I've done so later this afternoon. Thanks again, Mac.

~ Bal

**Comment by wordmuse** [comment]
OK - I created a video that demos what I'm seeing. A clarification: the problem doesn't happen with moving a node. It happens when copying a node, as will be seen in the video. As always I'll try to answer any questions you might have.

https://www.screencast.com/t/5Uprulayr

~ Bal

**Reply by macmarrum** [reply]
Thanks. This is clear now.

Congratulations. You've found a bug.

I'll open an issue for it.

**Reply by macmarrum** [reply]
Issue
- https://github.com/freeplane/freeplane/issues/616

**Reply by wordmuse** [reply]
OK - cool. Looking forward to having it fixed. 
Much appreciated.

~ Bal
